can/interfaces/pcan/pcan.py

```diff
@@ -86,9 +86,15 @@ def __init__(self, channel='PCAN_USBBUS1', state=BusState.ACTIVE, bitrate=500000
             Bitrate of channel in bit/s.
             Default is 500 kbit/s.
 
+        :param bool user_bit:
+            If true use the user defined bitrate with SJW, BRP, TSEG2, TSEG1
+            Default is False
         """
         self.channel_info = channel
-        pcan_bitrate = pcan_bitrate_objs.get(bitrate, PCAN_BAUD_500K)
+        if not user_bit:
+            pcan_bitrate = pcan_bitrate_objs.get(bitrate, PCAN_BAUD_500K)
+        else:
+            pcan_bitrate = TPCANHandle(bitrate)
```

Comment:
Its just a type definition:
TPCANHandle               = c_ushort    # Represents a PCAN hardware channel handle
But you are right I should use the :
TPCANBaudrate          = c_ushort    # Represents a PCAN Baud rate register value

"(i.e. simple 500000 does not translate directly to the proper BTR0BTR1 value for 500KBaud)"
In this case it does because we look the default values (most used) up in a dictionary 
line 55: pcan_bitrate_objs
In the docstring I describe it quite well I think:
            This argument can also be used for a user defined CAN-speed (e.g.: 0x0012 for 1.33 MHz)
            For details on how to write a user speed register (SJW, TSEG 1/2, BRP) see:
            http://www.bittiming.can-wiki.info (NXP SJA1000)
Please tell me what I could add for better understandability

Reply:
The problem I see is that `bitrate` is an integer representation of the bitrate (e.g. 500000). If user_bit is set, then line 97 becomes `pcan_bitrate = TPCANHandle(500000)` but in the dictionary, for 500K bitrate, `pcan_bitrate = TPCANBaudrate(0x1C) #or TPCANBaudrate(28)`. They both can't be right. 

The same issue comes up for the written example in the docstring. I'd expect `bitrate` to be 1330000 for 1.33M bitrate.

So the definition of what `bitrate` represents changes if `user_bit = True` or `user_bit = False`, and I don't think it's a good idea to do that.

Perhaps it would be better to define a new configuration type or calculate the register value of BTR0BTR1 within this method based on `bitrate`.

Reply:
Is this solved? Once it is, I'd merge this PR.

Reply:
Still discussing in Issue #538